### PR TITLE
Add more rules

### LINKED ISSUE #76: More Properties' order
- `text-decoration`
- `user-select`
- `vertical-align`
- `white-space`
- `opacity`

@kpman @jigsawye

### PR DESCRIPTION
**Pull request by xxhomey19**
close #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@
     'border-bottom',
     'border-left',
     'border-radius',
+    'border-top-right-radius',
+		'border-bottom-right-radius',
```
Comment (tw0517tw) [inline README.md]: 為啥多縮排?

Comment (jigsawye) [reply README.md]: 我也震驚

File: README.md [hunk 2]
```diff
@@ -78,12 +82,22 @@
     'font-size',
     'line-height',
     'text-align',
+    'vertical-align',
+    'text-decoration',
+    'white-space',
+    'text-shadow',
```
Comment (kpman) [inline README.md]: how about 
```
vertical-align
text-align
text-decoration
text-shadow
white-space
```

or

```
text-align
text-decoration
text-shadow
vertical-align
white-space
```

Comment (kpman) [reply README.md]: make `text-xxx` together
@xxhomey19 @jigsawye

**Pull request merged**